guides/rails_guides/markdown/renderer.rb

```diff
@@ -1,14 +1,22 @@
 # frozen_string_literal: true
 
+require "rouge"
+
+# Add more common shell commands
+Rouge::Lexers::Shell::BUILTINS += "|bin/rails|brew|bundle|gem|git|node|rails|rake|ruby|sqlite3|yarn"
+
 module RailsGuides
   class Markdown
     class Renderer < Redcarpet::Render::HTML
       cattr_accessor :edge, :version
 
       def block_code(code, language)
+        formatter = Rouge::Formatters::HTML.new
+        lexer = ::Rouge::Lexer.find_fancy(lexer_language(language), code)
```

Comment:
I think the `code` arg here is only used when [specifying `"guess"` as the language](https://github.com/rouge-ruby/rouge/blob/v3.20.0/lib/rouge/lexer.rb#L47-L53).

Reply:
I've removed the `code` arg